examples/gatsby/README.md

```diff
@@ -18,12 +18,6 @@ npm install
 npm run develop
 ```
 
-or:
-
-<!-- #default-branch-switch -->
-
-[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/mui-org/material-ui/tree/master/examples/gatsby)
```

Comment:
It's not working and Gatsby's popularity is low, trending downward, so probably not worth trying to fix this.